openai/api_resources/moderation.py

```diff
@@ -4,21 +4,23 @@
 
 
 class Moderation(OpenAIObject):
-    VALID_MODEL_NAMES: List[str] = ["text-moderation-stable", "text-moderation-latest"]
+    VALID_MODEL_NAMES: List[str] = [
+        "text-moderation-stable", "text-moderation-latest"]
 
     @classmethod
     def get_url(self):
         return "/moderations"
 
     @classmethod
-    def create(cls, input: Union[str, List[str]], model: Optional[str] = None):
+    def create(cls, input: Union[str, List[str]], model: Optional[str] = None,
+               api_key: Optional[str] = None):
         if model is not None and model not in cls.VALID_MODEL_NAMES:
             raise ValueError(
                 f"The parameter model should be chosen from {cls.VALID_MODEL_NAMES} "
-                f"and it is default to be None."
+                f"and it is default to be None.",
             )
 
-        instance = cls()
+        instance = cls(api_key=api_key)
```

Comment:
Can we only have this line?

Reply:
Do you mean removing other linting changes? I removed them now.